Source/Tx.Windows/PerfCounters/PerfCounterRealTimeProbe.cs

```diff
@@ -55,7 +55,10 @@ public void OnTimer(object state)
 
         public override void Dispose()
         {
-            _timer.Dispose();
+            if (_timer != null)
+            {
+                _timer.Dispose();
+            }
 
             base.Dispose();
```

Comment:
Although still something to apply it is not directly related my issue. In case of an invalid counter name the _query in the base class from ([here](https://github.com/Microsoft/Tx/blob/master/Source/Tx.Windows/PerfCounters/PerfCounteReader.cs#L88)) is already set due to the call to [this](https://github.com/Microsoft/Tx/blob/master/Source/Tx.Windows/PerfCounters/PerfCounterRealTimeProbe.cs#L16) line.

I've verified my changeset using this code:

    void Main()
    {
        var observable = PerfCounterObservable.FromRealTime(TimeSpan.FromSeconds(1), 
                new[] { "Bogus" });
	
        observable.Subscribe(pc => pc.Dump());
    }